### ISSUE TITLE
Standard DDS files can crash game

### ISSUE THREAD

**Issue by Txpple** [Beamdog/nwn-issues#609]
Per the wiki at https://nwn.wiki/display/NWN1/DDS and if i recall one of the patch notes, NWN supports standard DDS if it is BC1 BC3 BC4 and BC5.  When making a DDS in non-Bio format, the game hardcrashes when loading.

Cross linked to crash-analysis here: https://discord.com/channels/382306806866771978/1217908992794431649

**To Reproduce**

- Place the tga and dds and custom plc_a01 (overriding armoir to a UV Sphere) file in development.  Place the module in modules folder.
- Load module which has the above.  Game crashes.  
- Remove the DDS and confirm that TGA version loads fine.

The sample DDS was created with CompressionatorCLI using the following command: CompressonatorCLI.exe -fd BC1 textest01.tga textest01.dds.  Creating a similar BC1 with standard build of Crunch crashes, too.  

**Crash Dump**

- Windows: Attach the generated .log AND .dmp

**Specifics**

 -  Windows
 - Game build 36-12
 -  Steam
[nwmain-crash-1710442495.dmp](https://github.com/Beamdog/nwn-issues/files/14617151/nwmain-crash-1710442495.dmp)

 - Repro Rate 100%
 - Content overrides? Yes as above.
[dds.zip](https://github.com/Beamdog/nwn-issues/files/14617149/dds.zip)

**Comment by Finaldeath** [comment]
Have you tried crunch with the same file? Several tools that generate DDS files can generate ones that won't work due to newer or esoteric encodings.

**Comment by Txpple** [comment]
Yes same issue with base crunch (https://github.com/BinomialLLC/crunch).  

I'm using just the standard encodings like BC1 and BC3.

NWNCrunch does not crash, but it also does NWN header info on the dds.

**Comment by Txpple** [comment]
Per Symmetric: When you write .dds files, you should set the DDSD_CAPS and DDSD_PIXELFORMAT flags, and for mipmapped textures you should also set the DDSD_MIPMAPCOUNT flag. However, when you read a .dds file, you should not rely on the DDSD_CAPS, DDSD_PIXELFORMAT, and DDSD_MIPMAPCOUNT flags being set because some writers of such a file might not set these flags.

Basically binomial crunch and compressionator do not set those flags, causing in game crash.

Tested with Nvidia Texture Tools Exporter and it works fine per Jasperre's recommendation.

Leaving this issue open however as dev may need to decide whether or not they want to address the hard crash on DDS files w/out the above flags properly set.  See also this discussion link https://discord.com/channels/255017439371329537/467392390429278211/1322981949031059538
